charts/sftpgo/templates/deployment.yaml

```diff
@@ -38,6 +38,9 @@ spec:
             - sftpgo
             - serve
           env:
+            {{- with .Values.envVars }}
+            {{- toYaml . | nindent 12 }}
+            {{- end }}
```

Comment:
Leaving it above `env` is fine, but should be moved below the manually defined variables.

Reply:
makes sense. Please check @sagikazarmark